README.md

```diff
@@ -1,3 +1,165 @@
+7. Create a new Client. Select the option **Credentials**, and on the menu bar, select **Create credentials**, a dropdown menu will be displayed, then, select **OAuth Client ID** option. 
+8. Then, a select input will ask the type of Application type that will be used, select **Web application**, then, fill the form, and on **Authorized redirect URIs** introduce this URI:
+    https://<your_mattermost_instance>/plugins/com.mattermost.apps/apps/Drive/oauth2/remote/complete
+9. After the Client has been configured, on the main page of **Credentials**, on the submenu **OAuth 2.0 Client IDs** will be displayed the new Client and the info can be accessible whenever you need it.
+
+## Configuration
+
+1. First, you need to install the app in your current Mattermost instance (refer to [Installation](#installation)), the ``/drive`` command should be available.
+2. Go to [Google Cloud Dashboard](https://console.cloud.google.com/home/dashboard) and create a new project (refer to [Create a Google Cloud Project](#create-a-google-cloud-project)).
+3. Return to Mattermost. 
+4. As a super admin user, run the ``/drive configure`` command.
+5. In the configuration modal, enter your Client ID, Client Secret and Service Account.
+    - Client ID and Client Secret: Used to generate the link to let the user connect with their Google Drive account.
+    - Service Account: Is the way the app will get the credentials to manage webhook notifications
+
+
+# Admin guide
+
+## Slash commands
+- ``/drive configure:``: This command will enable the connect and disconnect commands, that will allow the user to link with their Google Drive account. Client ID and Client Secret will be asked.
+
+
+# End user guide
+
+## Get started
+
+## Use ``/drive`` commands
+
+- ``/drive help``: This command will show all current commands available for this application.
+- ``/drive connect``: This command will create a new link, after clicked it will be redirected to select the Google account that will be used to execute all the available actions.
+- ``/drive create [docs | slide | sheets]``: This command will display a new modal where the data will be asked. 
+    - Title: Name of the file to be created 
+    - Message: Optional, only if a file is shared on the channel 
+    - File Access: Choose to share with members on the channel, anyone with the link or keep it private.
+- ``/drive notifications [start | stop]``: This command will start or stop the notifications. After a user has executed ``/drive connect`` command, notifications will start on the private channel with the user.
+
+
+## Channel header bindings
+
+- ``Save file on Drive:``: This option is available in any post, but validates that the selected post has any files on it. If the post
+has any files, a new modal will appear to let the user select the files that would like to be uploaded. After the action is done, a new post will appear as part of the thread to let you notified of the action.
+
+# Development
+
```

Comment:
For Mattermost Cloud the app does not need to be deployed and can simply be installed via the Marketplace.

Reply:
This is for local deployment, in case we do not use neither AWS nor docker.
This should be removed?

Reply:
Does that mean the section is about a development environment?

Reply:
Yes. It is for developers that would need/want to do some changes on the code.
We commented this on the last meeting, to see what should and shouldn't be shown on the README file

Reply:
Then let's make sure that the documentation makes it clear it's about development.